airflow/www/views.py

```diff
@@ -860,6 +860,53 @@ def rendered(self):
             root=root,
             title=title)
 
+    @expose('/rendered_k8s')
```

Comment:
Use hyphen for this one too:
```suggestion
    @expose('/rendered-k8s')
```